client/src/components/App.js

```diff
@@ -1,10 +1,27 @@
 import React from 'react';
 
 class App extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      streamerId: 3
+    }
+  }
+
+  componentDidMount() {
+    const streamerId = this.props.streamerId || this.state.streamerId
+    fetch('/api/livestream/' + streamerId)
```

Comment:
airbnb eslint will want this as a template string `/api/livestream/${streamerId}`

Reply:
I noticed that eslint was not registering for some react files. I got to look into that. Thanks for reminding me.

Reply:
I found the bug. I need to put the no-plusplus in a string.